### ISSUE TITLE
[CIT Bug?] Removing the enchantment glint from a specific enchanted book also removes the glint from any item with the enchantment on it

### ISSUE THREAD

**Issue by Vaderman24** [sp614x/optifine#665]
I came across this while experimenting changing the color of enchantment glint for certain items. After a bit of research, I found out that you could use a pure black glint.png file to completely remove the effect. I did the testing on enchanted books and it worked fine, or so I thought. Although I got the result I was hoping for, it came at the cost of also removing the glint from items with the enchantment on it.

Imgur album of problem in game http://imgur.com/a/aGM7r

Properties file for Smite book and removing enchantment glint;
**Enchanted Smite Book**
`matchItems=enchanted_book`
`stackSize=1`
`tile=book_enchanted_smite`
`enchantmentIDs=17`
**Removes Enchantment Glint From Smite Book**
`type=enchantment`
`items=enchanted_book`
`enchantmentIDs=17`
`texture=enchanted_item_glint_blank`

**Comment by It-is-Me49** [comment]
Yeah, I also realized this. It is pretty annoying. Do you know if their planning on fixing it?
